rules/aws_resource_missing_tags.go

```diff
@@ -70,36 +144,79 @@ func (r *AwsResourceMissingTagsRule) Check(runner tflint.Runner) error {
 		// Special handling for tags on aws_autoscaling_group resources
 		if resourceType == "aws_autoscaling_group" {
 			err := r.checkAwsAutoScalingGroups(runner, config)
+			err = runner.EnsureNoError(err, func() error {
+				return nil
+			})
```

Comment:
There is no need to call `EnsureNoError` here as we are using function callbacks to handle errors.